lib/shakapacker/commands.rb

```diff
@@ -61,7 +61,7 @@ def packs
 
       packs = all_files - manifest_config - current_version
       packs.reject { |file| File.directory?(file) }.group_by do |path|
-        base, _, ext = File.basename(path).scan(/(.*)(-[\da-f]+)(\.\w+)/).flatten
+        base, _, ext = File.basename(path).scan(/(.*)(-[\da-f]+)((?:\.\w+)+)/).flatten
```

Comment:
pretty sure this is equivalent with slightly less characters:

```suggestion
        base, _, ext = File.basename(path).scan(/(.*)(-[\da-f]+)([.\w]+)/).flatten
```

Reply:
I wait for @tomdracz on this and then we merge.

Reply:
Makes sense and tested that it works. Updated and good to merge @justin808